.eslintrc.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
```

Comment:
i prefer an `.eslintrc` file. that it’s “deprecated” officially is irrelevant; they’ll never remove it.